app/build.gradle

```diff
@@ -151,7 +151,7 @@ dependencies {
     // OAuth2 client
     implementation project(path: ':appauth')
     // Please try to stay in sync with the versions used in the ics-openvpn module
-    implementation "androidx.appcompat:appcompat:1.1.0"
+    implementation "androidx.appcompat:appcompat:1.6.0-rc01"
```

Comment:
If you are using a non-stable version of the library for a reason (e.g. missing feature, important bugfix), add a comment why you have to use it. If for no reason, please stick to stable versions, since those are safer in general